backup-fetch.sh

```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-/usr/bin/wal-g backup-fetch $PGDATA $1
```

Comment:
we need still need this script to be able to restore from backup

Reply:
it should have been moved to src/main/docker/root/usr/bin, will add it back